cmd/main.go

```diff
@@ -966,7 +967,22 @@ func runOpenshiftReleaseSyncerIfNeeded(
 
 		failOnError(startupLeader.RunWithLeader(context.Background(), func() error {
```

Comment:
should we extract this (and possibly other if applicable) anonymous function(s) to the relevant package?

Reply:
What will be the advantage of this move? IMHO when you see the anonymous functions in main you get a better sense on what they are doing, also I don't think it should be part of `releasesources` package, because it is about the behavior of the output  of its use case.

Reply:
I agree, it would be better to extract it. As to destination, I think it is part of the functionality of main, hence we should extract it to a function out side the main function, but in the same package. WDYT ?

Reply:
No, I do not think so. Common it's a "socks drawer" right now, the issue is if we put everything there it soon will be a nightmare about circular dependencies.

We can create a package used to initialize stuff from main: this way the dependencies will be from such package, freeing main from complex dependencies.

maybe `internal/releasesources/startup` ? `internal/releasesources//init` ? WDYT?